src/js/tabs/map.js

```diff
@@ -1,76 +1,86 @@
-window.addEventListener('message', function (e) {
-  var mainWindow = e.source;
-  var result = '';
-  try {
-    switch(e.data.action){
-       case 'zoom_in':
-         var zoom = map.getZoom();
-         zoom++;
-         map.setZoom(zoom);
-         break;
-       case 'zoom_out':
-         var zoom = map.getZoom();
-         zoom--;
-         map.setZoom(zoom);
-         break;
-       case 'center':
-         map.setCenter(new google.maps.LatLng(e.data.lat, e.data.lon));
-         marker.setPosition( new google.maps.LatLng( e.data.lat, e.data.lon ) );
-         map.panTo( new google.maps.LatLng( e.data.lat, e.data.lon ) );
-    }
-  } catch (e) {
-    console.log('message error');
-  }
-});
+const DEFAULT_ZOOM = 16,
+      DEFAULT_LON = 10.001362,
```

Comment:
Not sure you want to tell everybody where you live. ;-)

Reply:
They are not mine, there were in the old version too... maybe is better to let them as zero, like iNav does...

Reply:
Changed to lon lat zero, zero